docopt.cpp

```diff
@@ -21,10 +21,10 @@
 #include <cassert>
 #include <cstddef>
 
-using namespace docopt;
+namespace docopt {
```

Comment:
Can you explain why you're doing this? I don't necessarily oppose it but would like to understand the justification. (Especially because you labeled this as something to do with GCC, which shouldn't be an issue).

One reason I like the "docopt::" in the definitions is that it catches bugs (if a function gets defined that didn't have a declaration).

Reply:
Ok, use previous style